### DISCUSSION TITLE
How to Prevent Photo Merging on PhotoPrism Map

### DISCUSSION THREAD

**Discussion by longester** [photoprism/photoprism#4400] [Q&A]
How to Prevent Photo Merging on PhotoPrism Map

Hi everyone,

I am using PhotoPrism and have encountered an issue with the map feature. Currently, when I view my photos on the map, they tend to merge together, especially when they are taken close to each other. This makes it difficult to view individual photos and their exact locations.

I would like to know if there is a way to prevent this merging behavior. Is there a setting or a configuration that I can adjust to display the photos separately, even when they are close together? Any tips or suggestions would be greatly appreciated!

**Comment by lastzero** [comment]
You can find more information about the cluster view and the problem it solves here:

- https://github.com/photoprism/photoprism/issues/1187

While it currently cannot be configured (e.g. to change the zoom level or cluster size), you can click on each cluster and then browse all pictures in the mosaic view below (or open them in a new tab on desktop browsers).

**Reply by namadori** [reply]
Hello, I'd like to know if there is a way to change the behavior of the cluster, maybe by changing some internal configuration file? For my usage (trying to show places visited during road trips) the clustering is a bit too much "aggressive", hiding the real itinerary under a few big groups.
